cmd/snap-exec/main.go

```diff
@@ -33,24 +33,33 @@ import (
 // for the tests
 var syscallExec = syscall.Exec
 
+// commandline args
+var opts struct {
+	Command string `long:"command" description:"use a different command like {stop,post-stop} from the app"`
+}
+
 func main() {
 	if err := run(); err != nil {
 		fmt.Printf("cannot snap-exec: %s\n", err)
 		os.Exit(1)
 	}
 }
 
-func run() error {
-	var opts struct {
-		Positional struct {
-			SnapApp string `positional-arg-name:"<snapApp>" description:"the application to run, e.g. hello-world.env"`
-		} `positional-args:"yes" required:"yes"`
-
-		Command string `long:"command" description:"use a different command like {stop,post-stop} from the app"`
+func parseArgs(args []string) ([]string, error) {
```

Comment:
wondering if it would be clearer if this returned (app string, appArgs []string, err error) directly